tests/run-make/rustdoc-test-args/Makefile

```diff
@@ -0,0 +1,10 @@
+include ../tools.mk
+
+# Test that rustdoc will properly load in a theme file and display it in the theme selector.
```

Comment:
Copy-pasted title :)

Reply:
Oups. ^^'

Reply:
Can this also be written as `rmake.rs` instead of introducing a new Makefile-based test?